server/src/logic/controllers/rncp/rncpController.js

```diff
@@ -48,12 +48,12 @@ class RncpController {
     };
   }
 
-  async findRncpFromCfd(cfd) {
-    const result = await FicheRncp.findOne({ cfds: { $in: [cfd] } });
+  async findRncpListFromCfd(cfd) {
+    const result = await FicheRncp.find({ cfds: { $in: [cfd] } }).lean();
```

Comment:
Si tu veux éviter le `map` au moment du retour, tu peux passer directement par distinct :slightly_smiling_face: 
```suggestion
    const result = await FicheRncp.distinct("code_rncp", { cfds: { $in: [cfd] } });
```

Reply:
yes bien vu, done here : https://github.com/mission-apprentissage/tables-correspondances/pull/214/commits/6f72e39fd9b273b8f5cc1f17fa2706c896c861f5